lib/payment_processor/clients/braintree/transaction.rb

```diff
@@ -30,19 +30,8 @@ def sale
               email: @user[:email]
             }
           )
-          process_response(transaction)
         end
-
-        def process_response(braintree_response)
-          puts "processing response"
-          if braintree_response.success?
-            { success: true, transaction_id: braintree_response.transaction.id }.to_json
-          else
-            { success: false, errors: [] }.to_json
-          end
-        end
-
       end
     end
   end
```

Comment:
Is the response a view, though? I understood that it returns an object that we process on the back-end as we wish (persisting the transactions in our own DB), and that we send a view to the controller as a part of that processing.